smokey/features/environment.py

```diff
@@ -42,6 +46,43 @@ def before_all(context):
     context.http = session
 
 
+def autoretry_scenario(scenario_or_outline):
```

Comment:
TODO: Add a note that this function has been accepted upstream under `behave.contrib`. When behave 1.2.6 is released we can use that instead.